package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

Since another pr was merged recently, we should bump this to 4.2.2
```suggestion
  "version": "4.2.2",
```